pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -311,11 +311,6 @@ func Write(
 		id := ulid.MustNew(ulid.Timestamp(now), ulid.DefaultEntropy()).String()
 		objectType, objectID := tupleUtils.SplitObject(tk.GetObject())
 
-		conditionName, conditionContext, err := marshalRelationshipCondition(tk.GetCondition())
```

Comment:
- [ ] todo: add test for NULL on condition and condition context when a tuple is deleted from the store.